### DISCUSSION TITLE
Plan for the next log4j-2 release?

### DISCUSSION THREAD

**Discussion by HannesWell** [apache/logging-log4j2#1788] [General]
Hello,

can you give an estimate when the next release for log4j-2.x will happen?
In https://github.com/apache/logging-log4j2/pull/1375#issuecomment-1473977592 it was mentioned that you try to release every three months.
I saw there is a release for log4j-3.0.0-alpha1 but unfortunately I cannot use that.

Thank you in advance.

**Comment by ppkarwasz** [comment]
Hi @HannesWell,

We are actively revamping the release process, which you can follow in issues #1777 and #1780. We should be able to finish it by the end of September.

You can also follow the list of PRs/issues that were linked with this release: https://github.com/apache/logging-log4j2/milestone/4

We would really love to have a micro-Eclipse RCP project with integration tests for this release, but there will probably be no time for that. You can contribute to the discussion on [this dev thread](https://lists.apache.org/thread/m7mckym5onzw04qm55464ngy0blqg7p4) if you want.

Summarizing: we thank you for you patience, but it seems that the more time we make for Log4j, the more time the project requires. ;-)

**Reply by HannesWell** [reply]
Thanks for the clarification.
I saw the issue you mentioned but didn't want to get off-topic there.

Round about end September sounds great.

> We would really love to have a micro-Eclipse RCP project with integration tests for this release, but there will probably be no time for that. You can contribute to the discussion on [this dev thread](https://lists.apache.org/thread/m7mckym5onzw04qm55464ngy0blqg7p4) if you want.

Yes that would be great. I'll have a look at the discussion, but unfortunately I'm not familiar with the frameworks you mentioned in your initial posting and currently don't have the time to learn them and therefore probably cannot contribute much.

**Comment by jvz** [comment]
We've been increasing the frequency of releases recently due to improvements in our release automation. In fact, we've made a couple releases since you posted this discussion!
